protected/tests/fixtures/init.php

```diff
@@ -22,6 +22,7 @@
 	$this->loadFixture("file_format");
 	$this->loadFixture("file_type");
 
+	$this->loadFixture("dataset_log");
```

Comment:
@pli888 @rija Load in `dataset_log` here, its columns seem more suitable for testing `dataset_log table`.